static/app/views/stories/index.tsx

```diff
@@ -15,13 +17,24 @@ type Props = RouteComponentProps<{}, {}, any, StoriesQuery>;
 
 export default function Stories({location}: Props) {
   const story = useStoriesLoader({filename: location.query.name});
+  const [searchTerm, setSearchTerm] = useState('');
 
   return (
     <Layout>
+      <div style={{gridArea: 'aside'}}>
```

Comment:
I wanna be annoying, and suggest that this use `display: flex; gap: ${space(2)};` instead of the margin on the Input.

Reply:
no comments on my inline styles are allowed!